src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts

```diff
@@ -174,4 +200,24 @@ export class EntryFieldsComponent implements OnInit {
   onTechnologyRemoved($event: string[]) {
     this.store.dispatch(new entryActions.UpdateEntryRunning({ ...this.newData, technologies: $event }));
   }
+
+
+  ngOnDestroy(): void {
+
+    if(this.exponentialGrowth){
+      this.loadActivitiesSubscribe.unsubscribe();
+      this.loadActiveEntrySubscribe.unsubscribe();
+      this.actionSetDateSubscribe.unsubscribe();
+    }
+
+    console.log('Entry Fields OnDestroy');
+  }
+
+  isFeatureToggleActivated() {
+     return this.featureManagerService.isToggleEnabledForUser('exponential-growth').pipe(
+        map((enabled) => {
+          return enabled === true ? true : false;
```

Comment:
Is a way to consume feature toggle observables, to avoid `null` values being evaluated to `false`. We commonly do it and I am in favor of that, but if you think we can make it better let us know.

Reply:
Not sure if this `enabled === true ? true : false;` is going to _avoid null values being evaluated to false_. In addition, I believe, the `isToggleEnabledForUser` method is not going to return null values.